src/Components/ServiceScene/Breakdowns/FieldsBreakdownScene.tsx

```diff
@@ -199,13 +247,28 @@ export class FieldsBreakdownScene extends SceneObjectBase<FieldsBreakdownSceneSt
                   </a>{' '}
                   if you think this is a mistake.
                 </Alert>
-              </div>
+              </GrotError>
             ),
           }),
         }),
       ],
     });
   }
+  private buildClearFiltersLayout(clearCallback: () => void) {
+    return new SceneReactObject({
+      reactNode: (
+        <GrotError>
+          <Alert title="" severity="info">
+            No labels match these filters.{' '}
+            <a className={emptyStateStyles.link} onClick={() => clearCallback()}>
```

Comment:
Anchor with no link is considered no bueno. Mind using a span that looks like a link or something similar?

Reply:
Yeah or just a button. good catch

Reply:
<img width="1728" alt="image" src="https://github.com/grafana/explore-logs/assets/109082771/7154ffb9-9812-4d51-a67d-1f3d66a2af9d">
How's this?